buildSrc/subprojects/plugins/src/main/kotlin/gradlebuild/publish-public-libraries.gradle.kts

```diff
@@ -137,9 +137,8 @@ fun Project.publishNormalizedToLocalRepository() {
         isCanBeResolved = false
         isCanBeConsumed = true
         isVisible = false
-        outgoing.artifact(mapOf(
-            "file" to localRepository.get().asFile, // This is not lazy
-            "builtBy" to localPublish
-        ))
+        outgoing.artifact(localRepository) {
```

Comment:
Similarly it looks like you could just use `outgoing.artifact(localPublish)` with the latest snapshot

Reply:
The publishing tasks do not define any outputs because they are never up-to-date and primarily exist to upload to an external repository.

Reply:
Right, thanks for the explanation.